t/utils/utils.go

```diff
@@ -70,7 +70,8 @@ func init() {
 	flag.Parse()
 
 	// set up logger
-	if err := logutils.OverrideRootLog(*logLevel, "", true); err != nil {
+	loggerEnabled := *logLevel != ""
```

Comment:
@adambabik yes it works with `-log=` or `-log=""`. we use the same in the `statusd` command. I just force pushed because in a rebase I lost a `:=`